src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -111,6 +123,46 @@ export class TestRunner extends NodejsEnvTestRunner {
         };
     }
 
+    private _handleRunExpectMatcher(
+        browser: Browser,
+        expectMatchers: ExpectWdioMatchers,
+    ): BrowserViteEvents[BrowserEventNames.runExpectMatcher] {
+        const { publicAPI: session } = browser;
+
+        return async (payload, cb): Promise<void> => {
```

Comment:
I saw this approach from wdio - https://github.com/webdriverio/webdriverio/blob/v8.36.0/packages/wdio-runner/src/browser.ts#L368-L412